code/extension/DataGridUtils/Templates/NET/Interface.cs

```diff
@@ -22,6 +22,6 @@ public interface IssDataGridDataConvert {
 		/// <param name="ssDataJSON">JSON with the data and metadata for the data grid</param>
 		void MssConvertData2JSON(object ssData, out string ssDataJSON);
```

Comment:
Let's remove this method if it's deprecated? The extension is only used internally.

Reply:
will do that on the next release